docs/development/heat_dissipation.md

```diff
@@ -1,11 +1,92 @@
+
+## Switching loss
+
+When a transistor is switching, the conducting channel has to build up. When the gate to source voltage is higher than the threshold voltage, the channels starts to form and the resistance $r_{ds}$ starts to fall. Until the channel has formed completly, the transistor acts as a ohmic resistance, the power loss in it can be obtained by
+
+$$P_{swit} = \frac{1}{2\cdot T_s} \cdot t_{on} \cdot U_{ds} \cdot I_{ds}$$
+
+where $T_s$ is the timespan for one switching process. The equivalent function with $f_s = \frac{1}{T_s}$ shows that the switching loss increases with the switching frequency:
+
+$$P_{swit} = \frac{1}{2} \cdot t_{on} \cdot U_{ds} \cdot I_{ds} \cdot f_s$$
+
+Both $U_{ds}$ and $I_{ds}$ are changing during the switching time. Figure 1 shows voltage and current during the process.
+
+<figure>
+    <switching-loss/>
+    <br/>
+<center>
+    <figcaption><b>Figure 1.</b> Current and voltage of the transistor during switching.
+    </figcaption>
+</center>
+</figure>
+
+At $t1$, the gate voltage reaches the threshold voltage and the drain-source current starts to flow. At $t2$ the current reaches its maximum and the drain voltage starts to fall. In this point, the switching power loss is at its maximum too. Until $t3$, the drain voltage drops as inner resistance $r_{ds}$ decreases and so $t_{on} = t_3 -t_1$. The same procedure in reverse happens when the MOSFET is switched off.
+
+The total power loss per duty cylce can be aggregated as
+
+$$P_{tot} = 2\cdot P_{swit} + P_{cond}$$
+
+## Heat dissipation
+
+To dissipate heat from the board, some design considerations have to be made. In general, heat can be transfered through convection, conduction and radiation where radiation is the least effective and only used in special cases. Dissipation is necessary to keep parts from overheating and especially semiconductors deteriorate faster when working under high temperatures.
+
+In many cases, conduction is used to transfer heat between parts, e.g. the MOSFET case and a heat sink while convection is used to transfer the heat to the environment and away from the component. The main factor for heat dissipation is the thermal resistance. This can be the connection from a MOSFET to a heat sink or the junction within the MOSFET to its case. It depends on the thermal conductivity of elements and connection between parts. The greater the surface of a connection, the lower the resistance, copper has a better conductivity thus lower thermal resistance than plastic etc. For many components, thermal resistance is given in the datasheet.
```

Comment:
```suggestion
In many cases, conduction is used to transfer heat between parts, e.g. the MOSFET case and a heat sink while convection is used to transfer the heat to the environment and away from the component. The main factor for heat dissipation is the thermal resistance. This can be the connection from a MOSFET to a heat sink or the junction within the MOSFET to its case. It depends on the thermal conductivity of elements and connection between parts. The greater the surface of a connection, the lower the resistance. For example, copper has a better conductivity and thus lower thermal resistance than plastic. For many electronic components, thermal resistance values are given in the datasheet.
```